src/Microsoft.ML.DataView/VBufferEditor.cs

```diff
@@ -26,12 +29,10 @@ public static class VBufferEditor
         /// Creates a <see cref="VBufferEditor{T}"/> using
         /// <paramref name="destination"/>'s values and indices buffers.
         /// </summary>
-        /// <param name="destination">
-        /// The destination buffer.
-        /// </param>
-        /// <param name="newLogicalLength">
-        /// The logical length of the new buffer being edited.
-        /// </param>
+        /// <param name="destination">The destination buffer. Note that the resulting <see cref="VBufferEditor{T}"/> is assumed to take ownership
```

Comment:
(nit) I find it easier to read XML comments when the wording is separate from the XML, like it is in the `summary` comment above.

```C#
        /// <param name="destination">
        /// The destination buffer. Note that the resulting <see cref="VBufferEditor{T}"/> is assumed to take ownership
        /// of this passed in object, and so whatever <see cref="VBuffer{T}"/> was passed in as this parameter should not be used again, since its
        /// underlying buffers are being potentially reused.
        /// </param>
```
```

vs

```C#
        /// <param name="destination">The destination buffer. Note that the resulting <see cref="VBufferEditor{T}"/> is assumed to take ownership
        /// of this passed in object, and so whatever <see cref="VBuffer{T}"/> was passed in as this parameter should not be used again, since its
        /// underlying buffers are being potentially reused.</param>
```

Formatting it this way forces the reader to parse the XML in order to find where the words start.

Reply:
Well, all right. Maybe just for the longer more discursive ones.